semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/DenotationOps.scala

```diff
@@ -113,6 +114,38 @@ trait DenotationOps { self: DatabaseOps =>
       if (config.overrides.isAll) gsym.overrides.map(_.toSemantic)
       else Nil
 
+    private def anns: (List[s.Annotation], List[g.Symbol]) = {
+      val buf = List.newBuilder[g.Symbol]
+      val ganns = gsym.annotations.filter { gann =>
+        gann.atp.typeSymbol != definitions.MacroImplAnnotation
+      }
+      val sanns = ganns.map { gann =>
+        val (sann, todo) = gann.toSemantic
+        todo.foreach(buf.+=)
+        sann
+      }
+      (sanns, buf.result)
+    }
+
+    private def acc: Option[s.Accessibility] = {
+      if (gsym.privateWithin == NoSymbol) {
+        if (gsym.isPrivateThis) Some(s.Accessibility(a.PRIVATE_THIS))
+        else if (gsym.isPrivate) Some(s.Accessibility(a.PRIVATE))
+        else if (gsym.isProtectedThis) Some(s.Accessibility(a.PROTECTED_THIS))
+        else if (gsym.isProtected) Some(s.Accessibility(a.PROTECTED))
+        else Some(s.Accessibility(a.PUBLIC))
+      } else {
+        val ssym = gsym.privateWithin.toSemantic.syntax
+        if (gsym.isProtected) Some(s.Accessibility(a.PROTECTED_WITHIN, ssym))
+        else Some(s.Accessibility(a.PRIVATE_WITHIN, ssym))
+      }
+    }
+
+    private def owner: m.Symbol = {
+      if (gsym.isSemanticdbGlobal) gsym.owner.toSemantic
```

Comment:
We should have a flag `-P:semanticdb:owners:all/none` to skip `owners`, I have no use for this in scalafix at the moment and I can imagine it has non-trivial impact on the semanticdb filesize.

Reply:
Will do.